src/pages/VenuePage/TemplateWrapper.tsx

```diff
@@ -16,6 +18,7 @@ import { PlayaRouter } from "components/templates/Playa/Router";
 import { ReactionPage } from "components/templates/ReactionPage";
 
 import { ChatSidebar } from "components/organisms/ChatSidebar";
+import UserProfileModal from "components/organisms/UserProfileModal";
```

Comment:
It would be good to use a named export and remove the default export from `UserProfileModal` (default exports are a legacy pattern we're trying to eradicate from the codebase).

Should be a fairly minimal change now that it should only be imported in one or a couple of locations.